src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/CSharpSyntaxClassificationService.cs

```diff
@@ -37,7 +37,8 @@ public CSharpSyntaxClassificationService(HostLanguageServices languageServices)
                     new OperatorOverloadSyntaxClassifier(),
                     new SyntaxTokenClassifier(),
                     new UsingDirectiveSyntaxClassifier(),
-                    new DiscardSyntaxClassifier()
+                    new DiscardSyntaxClassifier(),
+                    new FunctionPointerCallingConventionClassifier(),
```

Comment:
not a fan.  this should fallout from NameSyntaxClassifier :(

Reply:
@CyrusNajmabadi I think there is no "NameSyntax" nodes involved here. `NameSyntaxClassifier` operates on `NameSyntax` nodes.